backend/src/main/kotlin/hu/bme/sch/cmsch/model/UserEntity.kt

```diff
@@ -182,6 +182,8 @@ data class UserEntity(
 
 ): ManagedEntity, CmschUser {
 
+    var detailsImported: Boolean = false
```

Comment:
Save this property similar to the other ones. (property:GenerateInput, property:ImportFormat) You can use the `grantSellFood` property for reference.

Reply:
Did you mean to modify this like: 

`@property:GenerateOverview(visible = false)`
`@property:ImportFormat(ignore = false, columnId = 14)`
`var detailsImported: Boolean = false`

Of course inside the brackets, only these annotations are required? (the columnId is right this time)

Reply:
Yep! But make sure you use the appropriate type info for ImportFormat. 
Here is an example from GroupEntity:

```kotlin
    @JsonView(value = [ Edit::class ])
    @Column(nullable = false)
    @property:GenerateInput(type = INPUT_TYPE_SWITCH, order = 15, label = "Elhagyható", note = "A csoport tagjai megváltoztathatják a csoportjukat")
    @property:GenerateOverview(visible = false)
    @property:ImportFormat(ignore = false, columnId = 8, type = IMPORT_BOOLEAN)
    var leaveable: Boolean = false,
```

Sorry about the `grantSellFood` example I mentioned. It was removed long ago. :DD